opencti-platform/opencti-front/src/utils/Charts.js

```diff
@@ -626,6 +626,12 @@ export const radarChartOptions = (
   },
 });
 
+/**
+ * @param {Theme} theme
+ * @param {string[]} labels
+ * @param {function} formatter
+ * @param {string} legendPosition
+ */
```

Comment:
Just curious about that, can you explain to me what is the utility of it ? :)

Reply:
It allows IDE like Intellij to be able to check the types of the parameters even if it is a JS file when you use this function in a TS file